metadata.yml

```diff
@@ -0,0 +1,73 @@
+'@context': https://schema.org/
+creativeWorkStatus: Published
+type: LearningResource
+name: Annif Tutorial
+description: >-
+  Many libraries and related organizations are exploring automated methods for
+  metadata creation. This Annif Tutorial offers an introduction to the
+  multilingual automated indexing tool, Annif (annif.org), which can be
+  integrated into a library’s metadata production system. The tutorial provides
+  instructional videos and exercises for self-study purposes. Learners will gain
+  hands-on experience with Annif by setting it up, training its algorithms with
+  sample data, and generating subject suggestions for new documents. The
+  tutorial includes both basic and complex scenarios.
+license: https://creativecommons.org/licenses/by/4.0/deed.de
+id: https://github.com/NatLibFi/Annif-tutorial
+creator:
+  - givenName: Juho
+    familyName: Inkinen
+    id: https://orcid.org/0000-0002-6497-6171
+    type: Person
+    affiliation:
+      name: National Library of Finland
+      type: Organization
+  - givenName: Osma
+    familyName: Suominen
+    id: https://orcid.org/0000-0003-0042-0745
+    type: Person
+    affiliation:
+      name: National Library of Finland
+      type: Organization
+  - givenName: Mona
+    familyName: Lehtinen
+    id: https://orcid.org/0009-0004-1260-823X
+    type: Person
+    affiliation:
+      name: University of Turku
```

Comment:
University of Turku is not correct, should be "National Library of Finland" like for me and Osma. FYI @monalehtinen

Reply:
Also orcid should be https://orcid.org/0000-0002-4735-0214

Reply:
My apologies, I must have gotten the wrong profile. Fixed with 57a764e87b718328e0d7a467c9fafb75c8912c85